lib/helper.js

```diff
@@ -3,15 +3,14 @@ var React = require('react/addons');
 var TestUtils = React.addons.TestUtils;
 
 /**
- * Renders a component
+ * Renders a component. Method signature is the same as
+ * https://facebook.github.io/react/docs/top-level-api.html#react.createelement
  *
  * @method renderComponent
- * @param {ReactComponent} Component class
- * @param {Object} props used to initialize the component
```

Comment:
this documentation is helpful because it tells the use what params this method can take without he having to go to check react's doc...